dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/loadbalance/LeastActiveLoadBalance.java

```diff
@@ -70,7 +68,7 @@
         }
         if (!sameWeight && totalWeight > 0) {
             // If (not every invoker has the same weight & at least one invoker's weight>0), select randomly based on totalWeight.
-            int offsetWeight = ThreadLocalRandom.current().nextInt(totalWeight);
+            int offsetWeight = ThreadLocalRandom.current().nextInt(totalWeight) + 1;

```

Comment:
Why change `ThreadLocalRandom.current().nextInt(totalWeight)` to `ThreadLocalRandom.current().nextInt(totalWeight) + 1` ?

Reply:
Is it possible that +1 causes offset not equal or less 0?

Reply:
If not change to `+1`, that will cause a bug.
U can look at my unit test for more detail.